src/hyperlight_host/src/sandbox/outb.rs

```diff
@@ -93,6 +94,58 @@ pub(super) fn outb_log(mgr: &mut SandboxMemoryManager<HostSharedMemory>) -> Resu
     Ok(())
 }
 
+const ABORT_TERMINATOR: u8 = 0xFF;
+
+thread_local! {
+    static GUEST_ABORT_BUFFER: RefCell<Vec<u8>> = const { RefCell::new(Vec::new()) }
```

Comment:
Can we limit the size of this? Could potentially be a ddos target otherwise

Reply:
Will add a max length and check for it.